src/lwc/psElevateTokenHandler/psElevateTokenHandler.js

```diff
@@ -41,13 +42,39 @@ class psElevateTokenHandler {
     * @description Returns credit card tokenization Visualforce page URL
     */
     getTokenizeCardPageURL() {
-        const namespace = this.getCurrentNamespace();
-
-        return namespace
-            ? `/apex/${namespace}__${TOKENIZE_CARD_PAGE_NAME}`
+        return this.currentNamespace
+            ? `/apex/${this.currentNamespace}__${TOKENIZE_CARD_PAGE_NAME}`
             : `/apex/${TOKENIZE_CARD_PAGE_NAME}`;
     }
 
+    buildVisualForceOriginURLs(domainInfo) {
+        let url = `https://${domainInfo.orgDomain}--c.visualforce.com`;
```

Comment:
What if we changed the order of operations here? I don't like that we're doing string replacement to build these URLs when we're in a namespaced environment. Can we move `if (this.currentNamespace)` further up in this function, and build our URLs with the correct namespace instead of replacing `'--c'` in a string?

I think we need a function with a signature of `buildNamespacedURLs(domainInfo, namespace)`, which would replace the need for `convertToNamespacedURLs`